Purchases/Logging/Strings/OfferingStrings.swift

```diff
@@ -13,35 +13,121 @@
 //
 
 import Foundation
+import StoreKit
 
 // swiftlint:disable identifier_name
 enum OfferingStrings {
 
-    static let cannot_find_product_configuration_error = "Could not find SKProduct for %@ " +
-        "\nThere is a problem with your configuration in App Store Connect. " +
-        "\nMore info here: https://errors.rev.cat/configuring-products"
-    static let completion_handlers_waiting_on_products = "%lu completion handlers waiting on products"
-    static let fetching_offerings_error = "Error fetching offerings - %@"
-    static let sk_request_failed = "SKRequest failed: %@"
-    static let fetching_products_finished = "Products request finished."
-    static let fetching_products = "Requesting products from the store with identifiers: %@"
-    static let found_existing_product_request = "Found an existing request for products: %@, appending " +
-        "to completion"
-    static let invalid_product_identifiers = "Invalid Product Identifiers - %@"
-    static let list_products = "%@ - %@"
-    static let no_cached_offerings_fetching_from_network = "No cached Offerings, fetching from network"
-    static let no_cached_requests_and_products_starting_skproduct_request = "No existing requests and " +
-        "products not cached, starting SKProducts request for: %@"
-    static let offerings_stale_updated_from_network = "Offerings updated from network."
-    static let offerings_stale_updating_in_background = "Offerings cache is stale, updating from " +
-        "network in background"
-    static let offerings_stale_updating_in_foreground = "Offerings cache is stale, updating from " +
-        "network in foreground"
-    static let products_already_cached = "Skipping products request because products were already " +
-        "cached. products: %@"
-    static let retrieved_products = "Retrieved SKProducts: "
-    static let skproductsrequest_did_finish = "SKProductsRequest did finish"
-    static let skproductsrequest_received_response = "SKProductsRequest request received response"
-    static let vending_offerings_cache = "Vending Offerings from cache"
+    case cannot_find_product_configuration_error(identifiers: Set<String>)
+
+    case fetching_offerings_error(error: String)
+
+    case found_existing_product_request(identifiers: Set<String>)
+
+    case no_cached_offerings_fetching_from_network
+
+    case no_cached_requests_and_products_starting_skproduct_request(identifiers: Set<String>)
+
+    case offerings_stale_updated_from_network
+
+    case offerings_stale_updating_in_background
+
+    case offerings_stale_updating_in_foreground
+
+    case products_already_cached(identifiers: Set<String>)
+
+    case sk_request_failed(error: Error)
+
+    case skproductsrequest_did_finish
+
+    case skproductsrequest_received_response
+
+    case vending_offerings_cache
+
+    case retrieved_products(products: [SKProduct])
+
+    case list_products(inputOne: String, inputTwo: String)
```

Comment:
this key is not used and I couldn't determine what both inputs mean 😕

Reply:
This log entry was lost during the migration: https://github.com/RevenueCat/purchases-ios/blob/d0ac5b9b422207f7c710d04cb77899e4e77d9495/Purchases/Purchasing/RCStoreKitRequestFetcher.m

During the migration, the storekit request management was split into products requests and refresh receipt requests. 
The corresponding method for this log is now found here https://github.com/RevenueCat/purchases-ios/blob/swift_migration/Purchases/Purchasing/ProductsManager.swift#L72, and it's not using our logging 🤦 
This is my bad, the file was there from before the migration and it predates some of our logging stuff, and I forgot to integrate it afterwards. 

I think we should just have the params with the correct names (which would be `productIdentifier: String` and `product: SKProduct`, and then update the `ProductsManager` file to add the correct logging. I'm happy to update the `ProductsManager` as a part of #800

Reply:
the other unused strings are probably related to that as well

Reply:
Ok, I'm going to update the case then :) thanks for the clarification 👍🏼